consensus/polybft/common/bytes.go

```diff
@@ -0,0 +1,13 @@
+package common
+
+// LeftPadBytes zero-pads slice to the left up to length l.
+func LeftPadBytes(slice []byte, l int) []byte {
```

Comment:
This is a red flag and has to be replaced right now. This code is from `go-ethereum` and we **cannot** use anything from there.